frontend/packages/dev-console/src/components/pipelineruns/list-page/PipelineRunHeader.tsx

```diff
@@ -1,41 +1,42 @@
 import { sortable } from '@patternfly/react-table';
+import i18n from '@console/internal/i18n';
```

Comment:
Any reason for not using `t()`? `i18n` does not reflect language change until refresh, so I think it should be avoided if possible.

Reply:
It is not a component.

Reply:
Okay.